orm/src/main/java/org/hibernate/search/v6poc/entity/orm/model/impl/AbstractHibernateOrmTypeModel.java

```diff
@@ -114,9 +114,15 @@ public boolean isSubTypeOf(MappableTypeModel other) {
 						return getProperty( propertyName );
 					}
 					catch (PropertyNotFoundException e) {
+						// Error resolving the property through Hibernate internals
 						// Ignore this property
 						return (PojoPropertyModel<?>) null;
```

Comment:
Very minor but you have a cast here and not in the catch just below.

Reply:
Yeah, it the cast on the first return was required by the compiler, which doesn't know what the return type of the lambda is without it. If I put a cast on the second return, I get a warning telling me it's not necessary...